samples/test/util.py

```diff
@@ -222,6 +223,20 @@ def _create_run():
     fire.Fire(main)
 
 
+def simplify_proto_struct(data: dict) -> dict:
+    res = {}
+    for key, value in data.items():
```

Comment:
Do we have intValue here?

Reply:
not sure, we can add one when we have a sample